microsoft-edge/webview2/release-notes.md

```diff
@@ -34,6 +34,225 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
+	* [CoreWebView2SharedBuffer:: get_Buffer method](#)
+	* [CoreWebView2SharedBuffer:: get_FileMappingHandle method](#)
+
+##### [Win32/C++](#tab/win32cpp)
+
+* [ICoreWebView2_17 interface](#)
+	* [ICoreWebView2_17::PostSharedBufferToScript method](#)
+* [ICoreWebView2Environment12 interface](#)
+	* [ICoreWebView2Environment12:: CreateSharedBuffer method](#)
+* [ICoreWebView2Frame4 interface](#)
+	* [ICoreWebView2Frame4:: PostSharedBufferToScript method](#)
+* [ICoreWebView2SharedBuffer interface](#)
+	* [ICoreWebView2SharedBuffer:: OpenStream method](#)
+	* [ICoreWebView2SharedBuffer:: Close method](#)
+	* [ICoreWebView2SharedBuffer:: get_Size method](#)
+	* [ICoreWebView2SharedBuffer:: get_Buffer method](#)
+	* [ICoreWebView2SharedBuffer:: get_FileMappingHandle method](#)
+---
+
+*  The Permission API:
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+* [CoreWebView2Profile4 interface](#)
+	* [CoreWebView2Profile4::SetPermissionState method](#)
+	* [CoreWebView2Profile4::GetNonDefaultPermissionSettings method](#) 
+* [CoreWebView2PermissionSettingCollectionView interface](#)
+	* [CoreWebView2PermissionSettingCollectionView:: GetValueAtIndex method](#)
+	* [CoreWebView2PermissionSettingCollectionView:: get_Count method](#)
+* [CoreWebView2PermissionSetting interface](#)
+	* [CoreWebView2PermissionSettingCollectionView:: get_PermissionKind method](#)
+	* [CoreWebView2PermissionSettingCollectionView:: get_PermissionOrigin method](#)
+	* [CoreWebView2PermissionSettingCollectionView:: get_PermissionState method](#)
+* [CoreWebView2ControllerOptions2 interface](#)
+	* [CoreWebView2ControllerOptions2:: get_ScriptLocale method](#)
+	* [CoreWebView2ControllerOptions2:: put_ScriptLocale method](#)
+* [CoreWebView2SetPermissionStateCompletedHandler interface](#)
+* [CoreWebView2GetNonDefaultPermissionSettingsCompletedHandler interface](#)
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+* [CoreWebView2Profile4 interface](#)
+	* [CoreWebView2Profile4::SetPermissionState method](#)
+	* [CoreWebView2Profile4::GetNonDefaultPermissionSettings method](#) 
+* [CoreWebView2PermissionSettingCollectionView interface](#)
+	* [CoreWebView2PermissionSettingCollectionView:: GetValueAtIndex method](#)
+	* [CoreWebView2PermissionSettingCollectionView:: get_Count method](#)
+* [CoreWebView2PermissionSetting interface](#)
+	* [CoreWebView2PermissionSettingCollectionView:: get_PermissionKind method](#)
+	* [CoreWebView2PermissionSettingCollectionView:: get_PermissionOrigin method](#)
+	* [CoreWebView2PermissionSettingCollectionView:: get_PermissionState method](#)
+* [CoreWebView2ControllerOptions2 interface](#)
+	* [CoreWebView2ControllerOptions2:: get_ScriptLocale method](#)
+	* [CoreWebView2ControllerOptions2:: put_ScriptLocale method](#)
+* [CoreWebView2SetPermissionStateCompletedHandler interface](#)
+* [CoreWebView2GetNonDefaultPermissionSettingsCompletedHandler interface](#)
+
+##### [Win32/C++](#tab/win32cpp)
+* [ICoreWebView2Profile4 interface](#)
```

Comment:
I'm not finding ICoreWebView2Profile4 at:
https://github.com/MicrosoftDocs/webview2-win32-reference/tree/auto/110.0.1587.40/1-0-1587-40

I'm only finding ICoreWebView2Profile2 there:
https://github.com/MicrosoftDocs/webview2-win32-reference/blob/auto/110.0.1587.40/1-0-1587-40/icorewebview2profile2.md